nvidia_deeplearningexamples_waveglow.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+title: WaveGlow
+summary: WaveGlow model for generating speech from mel spectrograms (generated by Tacotron2)
+category: researchers
+image: nvidia_logo.png
+author: NVIDIA
+tags: [audio]
+github-link: https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/SpeechSynthesis/Tacotron2
+featured_image_1: no-image
+featured_image_2: no-image
+---
+
+```python
+import torch
+hub_model = torch.hub.load(github='nvidia/DeepLearningExamples', model='nvidia_waveglow')
+```
+will load the WaveGlow model pre-trained on [LJ Speech dataset](https://keithito.com/LJ-Speech-Dataset/)
+
+### Model Description
+
+The Tacotron 2 and WaveGlow model form a text-to-speech system that enables user to synthesise a natural sounding speech from raw transcripts without any additional prosody information. The Tacotron 2 model (also available via torch.hub) produces mel spectrograms from input text using encoder-decoder architecture. WaveGlow is a flow-based model that consumes the mel spectrograms to generate speech.
+
+### Example
+
+Here's a sample execution on a dummy input:
+
+```python
+import torch
+print('\nLoading waveglow model from torch.hub.')
+hub_model = torch.hub.load(github='nvidia/DeepLearningExamples', model='nvidia_waveglow')
+hub_model = hub_model.cuda()
+hub_model = hub_model.remove_weightnorm(hub_model)
+hub_model.eval()
+inp = torch.randn([1,80,300], dtype=torch.float32).cuda()
+with torch.no_grad():
+    out = hub_model.infer(inp)
+print('\nWaveglow model test output:')
+print(out.size())
+```
+
```

Comment:
add a visualization of the test output, or some other way to interpret the output

Reply:
changed strategy of this example to go from plain text to sound using the two torch.hub models in series